### ISSUE TITLE
Add chechbox "Refresh after" in script parameters

### ISSUE THREAD

**Issue by ruslan528** [gitextensions/gitextensions#7274]
I use script:
"Is powershell" checkbox enabled
Command: git checkout -B "master" "origin/master";git pull --rebase --progress "origin";git submodule update --init --recursive;exit;

Every time it completes I must click Refresh button manually. Please add ability to do this automatically.

**Comment by mstv** [comment]
I guess, you let run the script "in background", @ruslan528.
The grid is refreshed if and only if the script is run in the foreground.

@gerhardol, IIRC we have an option to automatically update submodules, don't we?
At least the confirmation can be turned off:
![grafik](https://user-images.githubusercontent.com/36601201/66707487-9598ff80-ed41-11e9-90fd-74364ac0c5be.png)

**Comment by RussKie** [comment]
> IIRC we have an option to automatically update submodules, don't we?

Yes, that's correct. We have a setting to update submodules on checkout.


I just looked through the code, and it looks like the setting is controlled by the checkbox shown on the screenshot above.
It is absolutely confusing as it **toggles submodules update**, and not just controls whether a confirmation is shown or not.

**Comment by ruslan528** [comment]
Why closed? 
@mstv, no, I didn't run the script "in background".

**Comment by RussKie** [comment]
I closed the issue for the following reasons:

* General - you chose to ignore the issue template that we have, and thus
have not provided all necessary information to triage the issue
* Specific to your use case - we have the desired functionality to update
submodules already implemented.

**Comment by ruslan528** [comment]
@russkie, but issue not about submodules! It's about refreshing ui with commits list after powershell script execution. Now I do it by pressing F5 manually.

**Comment by mstv** [comment]
> The grid is refreshed if and only if the script is run in the foreground.

Unfortunately, this was not the full truth. It was ignored for PowerShell scripts.

**Comment by mstv** [comment]
Even worse, GE never waits for PowerShell scripts to finish.

Workaround: Your script does not really use the PowerShell.
So you could turn off both `Run in background` and `Is PowerShell` and catenate the commands using `&&`.

**Comment by ruslan528** [comment]
> So you could turn off both `Run in background` and `Is PowerShell` and catenate the commands using `&&`.

@mstv Please describe how to catenate commands using &&. I'd tried several way with no success(

**Comment by mstv** [comment]
Let `cmd` run the actual commands:
![grafik](https://user-images.githubusercontent.com/36601201/67602863-d815ff80-f777-11e9-9d78-87064ff3d845.png)

can appear in the context menu:
![grafik](https://user-images.githubusercontent.com/36601201/67602933-ff6ccc80-f777-11e9-8db0-8610de7c8a29.png)

hotkey can be assigned:
![grafik](https://user-images.githubusercontent.com/36601201/67603188-7efa9b80-f778-11e9-98a3-9331eb68de05.png)
